portfolio/src/main/data_proccessing/processor.py

```diff
@@ -13,25 +13,37 @@
 stored within a database."""
 
 import pandas as pd
-import numpy as np
 
 # TODO(kofimeighan): Create a function that can iterate over all files within 
 # the raw_data directory
-RAW_DATA_FILEPATH = 'raw_data/police_killings_2013.csv'
-CLEAN_DATA_FILEPATH = 'clean_data/police_killings_2013.csv'
-COLUMNS_OF_INTEREST = ['name', 'race', 'date', 'address', 'city',
-                       'state', 'zipcode',]
-dataframe = pd.read_csv(RAW_DATA_FILEPATH, index_col=0)
-csv_columns = dataframe.columns
-columns_to_keep = []
-
-# TODO(kofimeighan): save the zipcode as an int not a long within the database
-for csv_column in csv_columns:
-    stripped_csv_column = csv_column.lower().replace("'", "")
-    for column_of_interest in COLUMNS_OF_INTEREST:
-        if column_of_interest.lower() in stripped_csv_column:
-            columns_to_keep.append(csv_column)
-            break
-
-dataframe = dataframe[columns_to_keep]    
-dataframe.to_csv(CLEAN_DATA_FILEPATH, index=False)
+RAW_TIME_PATH = 'raw_data/police_killings_2013.csv'
+CLEAN_TIME_PATH = 'clean_data/time_series.csv'
+TIME_COLUMNS = ['date', 'name']
+
+
+def load_csv_filter_and_frame(file_path, column_names):
```

Comment:
How about load_and_filter_csv()?

Reply:
I was trying to make it clear that it returns a frame, but I am happy to change it.